students/csimmons/lesson03/mailroom.py

```diff
@@ -62,8 +67,8 @@ def print_donorlist(all_info):
     info_row = '{dname:<20s}$ {total:>13,.2f} {gifts:^10d}  $ {avg:>12,.2f}'.format
     print(header1)
     print(header2)
-    for i in range(len(all_info)):
-        print(info_row(dname=all_info[i][0], total=all_info[i][1], gifts=all_info[i][2], avg=all_info[i][3]))
+    for idx, info in enumerate(all_info):
+        print(info_row(dname=all_info[idx][0], total=all_info[idx][1], gifts=all_info[idx][2], avg=all_info[idx][3]))
```

Comment:
you also don't need enumerate here: if you remove it, you'd just delete all of the `idx` indexing:
```
for info in all_info:
    print(info_row(dname=all_info[0])
```
another trick is you can expand your vars as you're looping:
```
for name, total, gifts, avg in all_info: 
    print(info_row(dname=name, total=total....
```
which makes your code even more cleaner/readable

Reply:
That last trick (expanding the vars ) is very cool.  I'll be using it in the future.  Also, will address the .gitignore issue now

Reply:
Added this trick to both mailroom files - pretty cool. I have no idea why I didnt know you could do this...

Reply:
glad it worked out for you!